controllers/metal3.io/baremetalhost_controller.go

```diff
@@ -465,10 +492,7 @@ func clearRebootAnnotations(host *metal3v1alpha1.BareMetalHost) (dirty bool) {
 // which means we don't inspect even in Inspecting state
 func inspectionDisabled(host *metal3v1alpha1.BareMetalHost) bool {
 	annotations := host.GetAnnotations()
-	if annotations[inspectAnnotationPrefix] == "disabled" {
-		return true
-	}
-	return false
+	return annotations[inspectAnnotationPrefix] == "disabled"
 }
```

Comment:
Not blocking: it could be a good occasion to move this entire block into its own function, to reduce a little bit the size of the main reconcile loop

Reply:
hmm, that's already an independent function and not part of the reconcile(), no?

Reply:
That's a good point @andfasano . Would you mind if I address this an a follow up patch not to block the initial implementation one?

Reply:
Sounds fine for me